includes/class-sensei-learner-profiles.php

```diff
@@ -35,9 +35,28 @@ public function __construct() {
 		add_action( 'sensei_learner_profile_info', array( $this, 'learner_profile_courses_heading' ), 30, 1 );
 
 		// Add class to body tag
-		add_filter( 'body_class', array( $this, 'learner_profile_body_class' ), 10, 1 );
+		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
 	} // End __construct()
 
+	/**
+	 * Enqueue frontend JavaScripts.
+	 *
+	 * @since  1.0.0
+	 * @return void
```

Comment:
`@return void` is not necessary as per the [coding guidelines](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods).